src/PlaywrightSharp.Abstractions/SetNetworkCookieParam.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace PlaywrightSharp
+{
+    /// <summary>
+    /// Cookie data.
+    /// </summary>
+    /// <seealso TODOcref="Page.SetContentAsync(string, NavigationOptions)"/>
+    /// <seealso TODOcref="Page.DeleteCookieAsync(SetNetworkCookieParam[])"/>
+    /// <seealso TODOcref="Page.GetCookiesAsync(string[])"/>
+    public class SetNetworkCookieParam
```

Comment:
Let's call this NetworkCookie

Reply:
in playwright there are 2 interfaces - `NetworkCookie` & `SetNetworkCookieParam` - https://github.com/kblok/playwright/blob/master/src/network.ts